scripts/hudson/narayana.sh

```diff
@@ -704,13 +712,13 @@ function add_qa_xargs {
 
 function qa_tests_once {
   echo "QA Test Suite $@"
-  
-  
+
+
```

Comment:
If you are changing white space then why not the whole of the extra line?

Reply:
This is my IDE doing the work for me. I'll remove the extra line manually. Thanks for spotting this Michael

Reply:
You can change the IDE settings to not update files without your knowledge. It also makes the reviewers job that little bit easier (since it becomes clear what the functional changes are). We have said in the past that reformatting changes should go into a separate commit (but I generally ignore them if there aren't many since a commit with just reformatting makes the git commit history noisy; there's pros and cons).